roles/matrix-backup-borg/templates/config.yaml.j2

```diff
@@ -0,0 +1,37 @@
+#jinja2: lstrip_blocks: "True", trim_blocks: "True"
+
+location:
+  source_directories:
+  - {{ matrix_backup_borg_source }}
+  repositories:
+  - {{ matrix_backup_borg_repository }}
+  one_file_system: true
+  exclude_patterns:
+  {% for pattern in matrix_backup_borg_exclude %}
+  - {{ pattern }}
+  {% endfor %}
+
+storage:
+  compression: {{ matrix_backup_borg_compression }}
+  ssh_command: {{ matrix_backup_borg_ssh_command }}
+  archive_name_format: '{{ matrix_backup_borg_name }}'
+  encryption_passphrase: {{ matrix_backup_borg_passphrase }}
+
+retention:
+  keep_hourly: {{ matrix_backup_borg_hourly }}
```

Comment:
`matrix_backup_borg_hourly` -> `matrix_backup_borg_retention_keep_hourly`?

Likewise for the others?